galasa-managers-parent/galasa-managers-workflow-parent/dev.galasa.githubissue.manager/src/main/java/dev/galasa/githubissue/internal/GitHubIssueManagerImpl.java

```diff
@@ -170,7 +177,7 @@ public Result endOfTestMethod(@NotNull GalasaMethod galasaMethod, @NotNull Resul
 		return newResult;
 	}
 	
-	private Result overrideMethodResult(@NotNull GalasaMethod method, @NotNull Result currentResult, Throwable currentException) throws GitHubIssueManagerException {
+	private Result overrideMethodResult(@NotNull GitHubIssue gitHubIssue, @NotNull GalasaMethod method, @NotNull Result currentResult, Throwable currentException) throws GitHubIssueManagerException {
 
 		if (!currentResult.isFailed()) {
 			return null;
```

Comment:
Can we not say if(currentResult.isPassed) rather than the negation of a negative?

Reply:
Hi Will, we don't use currentResult.isPassed here we want to override a method's result only if it is Failed.  For example there are options for a method's result to be Ignored or EnvFail and these should not go through the overriding logic, so should return null here as they are not Failed. Hope that helps

Reply:
Yup - sorry mis-read